Source/Plugins/Core/com.equella.core/js/tsrc/search/SearchPage.tsx

```diff
@@ -78,6 +78,14 @@ import {
 } from "./components/SearchResultList";
 import StatusSelector from "./components/StatusSelector";
 
+// destructure strings import
+const searchStrings = languageStrings.searchpage;
```

Comment:
for consistency with line 87
```suggestion
const { searchpage: searchStrings } = languageStrings;
```

Reply:
Applied in 2bb8382